kong/tools/http.lua

```diff
@@ -527,4 +527,28 @@ do
 end
 
 
+do
+  local var = ngx.var
+  local get_headers = ngx.req.get_headers
```

Comment:
let's not localize this as I am fearing that it might skip these in some weird cases:
https://github.com/Kong/kong/blob/master/kong/globalpatches.lua#L118-L237

Reply:
done.